### ISSUE TITLE
Bloxstrap won't start correctly due to a previous setting

### ISSUE THREAD

**Issue by jvrring** [pizzaboxer/bloxstrap#42]
Running Bloxstrap v1.5.2.
This error pops up, I believe if you have the RFU auto close option set to false. It triggers when your browser launches the Bloxstrap client while RFU is open beforehand.

![image](https://user-images.githubusercontent.com/67696935/201183857-71602fcb-eab3-48fc-bb34-00a7a72a7c4d.png)

Here's my settings.json file:
```json
{
  "Channel": "LIVE",
  "VersionGuid": "version-d780cbcde4ab4f52",
  "CheckForUpdates": true,
  "BootstrapperStyle": 3,
  "BootstrapperIcon": 0,
  "Theme": 2,
  "UseDiscordRichPresence": true,
  "HideRPCButtons": true,
  "RFUEnabled": true,
  "RFUAutoclose": false,
  "UseOldDeathSound": true,
  "UseOldMouseCursor": false
}

**Comment by pizzaboxer** [comment]
That makes sense since RFU can't be updated while it's still running. Check in the system tray/task manager to see if it's still there. I'll fix this in a future update, but probably not now as I've just already released one, won't happen for most users, and is practically a once-every-six-months thing that happens.

**Comment by jvrring** [comment]
> That makes sense since RFU can't be updated while it's still running. Check in the system tray/task manager to see if it's still there. I'll fix this in a future update, but probably not now as I've just already released one, won't happen for most users, and is practically a once-every-six-months thing that happens.

Understandable. I'll just set `RFUAutoClose` to true for the time being.

**Comment by pizzaboxer** [comment]
...and I'm gonna have to release a new update anyway. lol. it'll be fixed in v1.5.3.

**Comment by pizzaboxer** [comment]
Just released v1.5.3, which has fixed this issue.
